### PR TITLE
[DOC] Minor text changes from doc review

### PR DESCRIPTION
**Pull request by PaulRMellor**
### Type of change

_Select the type of your PR_

- Documentation

### Description

Minor text changes from a review of part of the doc.

### Checklist

_Please go through this checklist and make sure all applicable tasks have been done_

- [ ] Update/write design documentation in `./design`
- [ ] Write tests
- [ ] Make sure all tests pass
- [x] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md

### CODE REVIEW

File: documentation/book/ref-kafka-authentication.adoc [hunk 1]
```diff
@@ -6,17 +6,18 @@
...
 
 Authentication is configured as part of the xref:assembly-configuring-kafka-broker-listeners-{context}[listener configuration] in the `authentication` property.
-When the `authentication` property is missing, no authentication will be enabled on given listener.
-The authentication mechanism which will be used is defined by the `type` field.
+The authentication mechanism is defined by the `type` field.
 
-The supported authentication mechanisms are: 
+When the `authentication` property is missing, no authentication is enabled on a given listener.
+
+Supported authentication mechanisms:
```
Comment (scholzj) [inline documentation/book/ref-kafka-authentication.adoc]:
```suggestion
Supported authentication mechanisms are:
```

Feel free to ignore this ... but it sounds better to me with the `are` added.

File: documentation/book/ref-kafka-rack.adoc [hunk 2]
```diff
@@ -6,9 +6,8 @@
 = Kafka rack awareness
 
 Kafka rack awareness can be configured in the `rack` property of `Kafka.spec.kafka`.
-The `rack` object has one mandatory field named `topologyKey`.
-This key needs to match one of the labels assigned to the {ProductPlatformName} cluster nodes.
-The label is used by {ProductPlatformName} when scheduling the Kafka broker pods to nodes.
+The `rack` object has one mandatory field named `topologyKey`, which needs to match one of the labels assigned to the {ProductPlatformName} cluster nodes. The label is used by {ProductPlatformName} when scheduling the Kafka broker pods to nodes.
```
Comment (scholzj) [inline documentation/book/ref-kafka-rack.adoc]:
```suggestion
The `rack` object has one mandatory field named `topologyKey`, which needs to match one of the labels assigned to the {ProductPlatformName} cluster nodes.
The label is used by {ProductPlatformName} when scheduling the Kafka broker pods to nodes.
```

File: documentation/book/ref-resource-limits-and-requests.adoc [hunk 3]
```diff
@@ -24,10 +24,11 @@ Reserving the resources will ensure that they are always available.
 
 IMPORTANT: If the resource request is for more than the available free resources in the {ProductPlatformName} cluster, the pod will not be scheduled.
 
-Resource requests can be specified in the `request` property.
-The resource requests currently supported by {ProductName} are memory and CPU.
-Memory is specified under the property `memory`.
-CPU is specified under the property `cpu`.
+Resources for requests are specified in the `requests` property.
```
Comment (scholzj) [inline documentation/book/ref-resource-limits-and-requests.adoc]:
```suggestion
Resources requests are specified in the `requests` property.
```

File: documentation/book/ref-resource-limits-and-requests.adoc [hunk 4]
```diff
@@ -24,10 +24,11 @@ Reserving the resources will ensure that they are always available.
...
+Resources currently supported by {ProductName}:
```
Comment (scholzj) [inline documentation/book/ref-resource-limits-and-requests.adoc]:
```suggestion
Resources requests currently supported by {ProductName}:
```

File: documentation/book/assembly-kafka-connect-authentication.adoc [hunk 5]
```diff
@@ -12,10 +12,10 @@
 = Connecting to Kafka brokers with Authentication
 
 By default, Kafka Connect will try to connect to Kafka brokers without any authentication.
-Authentication can be enabled in the `KafkaConnect` and `KafkaConnectS2I` resources.
+Authentication is enabled through configuration of the `KafkaConnect` and `KafkaConnectS2I` resources.
```
[the remark this answers is not in the document]
Comment (PaulRMellor) [reply documentation/book/assembly-kafka-connect-authentication.adoc]:
```suggestion
Authentication is enabled through the `KafkaConnect` and `KafkaConnectS2I` resources.
```

File: documentation/book/assembly-kafka-mirror-maker-authentication.adoc [hunk 6]
```diff
@@ -11,10 +11,10 @@
 = Connecting to Kafka brokers with Authentication
 
 By default, Kafka Mirror Maker will try to connect to Kafka brokers without any authentication.
-Authentication can be enabled in the `KafkaMirrorMaker` resource.
+Authentication is enabled is enabled through configuration of the `KafkaMirrorMaker` resource.
```
[the remark this answers is not in the document]
Comment (PaulRMellor) [reply documentation/book/assembly-kafka-mirror-maker-authentication.adoc]:
```suggestion
Authentication is enabled is enabled through the `KafkaMirrorMaker` resource.
```

File: documentation/book/assembly-storage.adoc [hunk 7]
```diff
@@ -8,9 +8,13 @@
 :parent-context: {context}
 
 [id='assembly-storage-{context}']
-= Kafka and Zookeeper storage
+= Kafka and Zookeeper storage types
```
[the remark this answers is not in the document]
Comment (PaulRMellor) [reply documentation/book/assembly-storage.adoc]: If you want to know the supported storage types you can search and find it here where they are listed. Then you dive deeper.

File: documentation/book/assembly-kafka-connect-authentication.adoc (hunk 5 above)
[the remark this answers is not in the document]
Comment (scholzj) [reply documentation/book/assembly-kafka-connect-authentication.adoc]: Yeah, that would work for me.

File: documentation/book/ref-kafka-authentication.adoc (hunk 1 above)
Comment (PaulRMellor) [reply documentation/book/ref-kafka-authentication.adoc]: When used with a list the colon is the 'are', so I don't think we need it.

File: documentation/book/assembly-kafka-mirror-maker-authentication.adoc (hunk 6 above)
[the remark this answers is not in the document]
Comment (PaulRMellor) [reply documentation/book/assembly-kafka-mirror-maker-authentication.adoc]:
```suggestion
Authentication is enabled through the `KafkaMirrorMaker` resource.
```

File: documentation/book/assembly-healthchecks.adoc [hunk 8]
```diff
@@ -12,8 +12,8 @@
 
 = Healthchecks
 
-Healthchecks are periodical tests which verify that the application's health.
-When the Healthcheck fails, {ProductPlatformName} can assume that the application is not healthy and attempt to fix it.
+Healthchecks are periodical tests which verify the health of an application.
+When a Healthcheck probe fails, {ProductPlatformName} assumes that the application is not healthy and attempts to fix it.
```
Comment (laidan6000) [inline documentation/book/assembly-healthchecks.adoc]: Sounds good. I also edited this sentence but on a different branch! Please use your version when resolving the merge conflict.

File: documentation/book/assembly-storage.adoc [hunk 9]
```diff
@@ -8,9 +8,13 @@
...
 
-As stateful applications, Kafka and Zookeeper need to store data on disk. {ProductName} supports three different types of storage for this data: ephemeral, persistent, and JBOD storage.
+As stateful applications, Kafka and Zookeeper need to store data on disk. {ProductName} supports three storage types for this data:
+
+* Ephemeral
+* Persistent
+* JBOD storage.
```
Comment (ncbaratta) [inline documentation/book/assembly-storage.adoc]: If you're going to have a period here then the three before should be formatted as if they were in a sentence - like

* Ephemeral,
* Persistent,
* and JBOD storage.

File: documentation/book/assembly-healthchecks.adoc (hunk 8 above)
Comment (PaulRMellor) [reply documentation/book/assembly-healthchecks.adoc]: Will do.

File: documentation/book/assembly-storage.adoc (hunk 9 above)
Comment (PaulRMellor) [reply documentation/book/assembly-storage.adoc]: Will remove it.

**Pull request merged**